package.xml

```diff
@@ -22,6 +22,7 @@
   <test_depend>ament_cmake_nose</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>demo_nodes_cpp</test_depend>
   <test_depend>diagnostic_msgs</test_depend>
```

Comment:
Absolutely right: 959cd613e08d689b0d7542996971a8ef09342a80

The CI build should pass though since packaging job doesn't build with isolation.